Jenkinsfile

```diff
@@ -24,40 +24,11 @@ node ('rocmtest')
     deleteDir( )
```

Comment:
@scchan could you assess if arguments from @pfultz2 is valid? I don't have way to measure time required on the jenkins job. only you guys have..